### ISSUE TITLE
Make relative url configurable 

### ISSUE THREAD

**Issue by xdxxxdx** [Informatievlaanderen/VSDS-LDESServer4J#894]
**Details of the feature requests**
Follow up the discussion in slack: https://vsdscore.slack.com/archives/C03CQKA4RU7/p1692864439127089
It will be useful to enable the relative URL configurable.

**Current solution**
I tried to configure the LDES server with:

```
mongock:
    migration-scan-package: VSDS
springdoc:
    swagger-ui:
        path: /v1/swagger
ldes-server:
    host-name: ""
management:
    tracing:
        enabled: false
spring:
  data:
    mongodb:
        database: ldes
        host: ldes-mongodb
        port: 27017
```

Relative url fragmentation is return 
```
@prefix kbo:   </kbo/> .
@prefix ldes:  <https://w3id.org/ldes#> .
@prefix legal: <http://www.w3.org/ns/legal#> .
@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix shacl: <http://www.w3.org/ns/shacl#> .
@prefix terms: <http://purl.org/dc/terms/> .
@prefix tree:  <https://w3id.org/tree#> .

<http://localhost:8080/kbo/shape>
        rdf:type           shacl:NodeShape ;
        shacl:targetClass  legal:legalEntity .

</kbo>  rdf:type            ldes:EventStream ;
        ldes:timestampPath  terms:issued ;
        ldes:versionOfPath  terms:isVersionOf ;
        tree:view           kbo:by-name .

kbo:by-name  rdf:type  tree:Node ;
        tree:relation  [ rdf:type   tree:Relation ;
                         tree:node  </kbo/by-name?substring=>
                       ] .
```
But the fragmentation cannot be reached
```
C:\Users\XXDE>curl http://localhost:8080/kbo/by-name?substring=
Index -1 out of bounds for length 0
```

**Comment by pj-cegeka** [comment]
This feature is implemented as of: #1034

**Comment by xdxxxdx** [comment]
Also validated with Ldes server Snapshot 2.5.0
